CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+# 17.0.0
+ * add support for Angular 17
```

Comment:
In this pull request, we are only pulling in the work to support Angular 17, the other releases are not ported yet.

I would not want to create a confusing history of changes in this file, so for now, I would roll back the changes and only mention Angular 17.

@danisss9: Can you push this change to the pull request please? Once this is done, I will merge the pull request and then pull in the rest of my changes from ng17 branch.

Reply:
done